README.md

```diff
@@ -8,37 +8,39 @@ Currently covered:
+| GobRPC_1K-4      | 100000 |   63812 ns/op | 32.09 MB/s |   4744 B/op |  33 allocs/op |
+| GobRPC_64K-4     |   5000 | 1414237 ns/op | 92.68 MB/s | 263935 B/op |  97 allocs/op |
+| ProtoRPC_1K-4    | 200000 |   56618 ns/op | 36.17 MB/s |   2632 B/op |  27 allocs/op |
+| ProtoRPC_64K-4   |   5000 | 1413800 ns/op | 92.71 MB/s | 280219 B/op |  93 allocs/op |
+| ProtoHTTP1_1K-4  |  20000 |  388989 ns/op |  5.26 MB/s |  57736 B/op | 436 allocs/op |
+| ProtoHTTP1_64K-4 |   3000 | 2001305 ns/op | 65.49 MB/s | 823248 B/op | 345 allocs/op |
+| ProtoHTTP2_1K-4  |  50000 |  165243 ns/op | 12.39 MB/s |  89641 B/op | 107 allocs/op |
+| ProtoHTTP2_64K-4 |   5000 | 1804282 ns/op | 72.64 MB/s | 944128 B/op | 330 allocs/op |
+
+## go version devel +51586aa Thu Oct 29 19:27:47 2015 +0000 darwin/amd64
 
 These are faster than go1.5.1 thanks to cloudflare's amd64 assembly implementation of
 AES-GCM: https://github.com/golang/go/commit/efeeee3.
 
-| Bench          | Iters  | Speed         | Throughput  | Allocations | Allocations    |
-|----------------|-------:|--------------:|------------:|------------:|---------------:|
-| GRPC_1K        | 100000 |   70215 ns/op |  29.17 MB/s |  16658 B/op |  114 allocs/op |
-| GRPC_64K       |  10000 |  768996 ns/op | 170.45 MB/s | 714502 B/op |  357 allocs/op |
-| GobRPC_1K      | 200000 |   39776 ns/op |  51.49 MB/s |   4744 B/op |   33 allocs/op |
-| GobRPC_64K     |  20000 |  334447 ns/op | 391.91 MB/s | 263858 B/op |   97 allocs/op |
-| ProtoRPC_1K    | 200000 |   32926 ns/op |  62.20 MB/s |   2632 B/op |   27 allocs/op |
-| ProtoRPC_64K   |  20000 |  341672 ns/op | 383.62 MB/s | 280175 B/op |   93 allocs/op |
-| ProtoHTTP1_1K  |  20000 |  372478 ns/op |   5.50 MB/s |  59098 B/op |  448 allocs/op |
-| ProtoHTTP1_64K |  10000 | 1414341 ns/op |  92.67 MB/s | 931481 B/op | 1091 allocs/op |
-
-golang.org/x/net/http2 currently doesn't implement sending requests with a body, so those benchmarks fail.
+| Bench            | Iters  | Speed         | Throughput  | Allocations | Allocations    |
+|------------------|-------:|--------------:|------------:|------------:|---------------:|
+| GRPC_1K-4        | 100000 |   74513 ns/op |  27.48 MB/s |  16055 B/op	|  106 allocs/op |
+| GRPC_64K-4       |  10000 |  858855 ns/op | 152.61 MB/s | 714106 B/op	|  346 allocs/op |
+| GobRPC_1K-4      | 200000 |   40553 ns/op |  50.50 MB/s |   4745 B/op	|   33 allocs/op |
+| GobRPC_64K-4     |  20000 |  421501 ns/op | 310.96 MB/s | 263899 B/op	|   97 allocs/op |
+| ProtoRPC_1K-4    | 200000 |   33851 ns/op |  60.50 MB/s |   2632 B/op	|   27 allocs/op |
+| ProtoRPC_64K-4   |  20000 |  479764 ns/op | 273.20 MB/s | 280209 B/op	|   93 allocs/op |
+| ProtoHTTP1_1K-4  |  30000 |  332801 ns/op |   6.15 MB/s |  52926 B/op	|  399 allocs/op |
+| ProtoHTTP1_64K-4 |   5000 | 1419998 ns/op |  92.30 MB/s | 929968 B/op	| 1080 allocs/op |
+| ProtoHTTP2_1K-4  |  50000 |  128962 ns/op |  15.88 MB/s |  89091 B/op	|  107 allocs/op |
```

Comment:
Parts of it are: https://github.com/grpc/grpc-go/blob/174192fc93efcb188fc8f46ca447f0da606b6885/transport/http2_server.go#L46

But grpc doesn't use `net/http`; it builds its own implementation off the lower-level interfaces (line 211 in the above link). So apparently `net/http` has some higher per-request overhead, while grpc has more per-byte overhead in larger requests.